priv/templates/extended_bin

```diff
@@ -11,31 +11,30 @@ set -e
 # when writing log message to terminal, leaving the heart process unable
 # to do its work.
 RUN_ERL_DISABLE_FLOWCNTRL=${RUN_ERL_DISABLE_FLOWCNTRL:-true}
-export $RUN_ERL_DISABLE_FLOWCNTRL
+export RUN_ERL_DISABLE_FLOWCNTRL
 
-if [ "$TERM" = "dumb" -o -z "$TERM" ]; then
+if [ "$TERM" = "dumb" ] || [ -z "$TERM" ]; then
   export TERM=screen
 fi
 
 # OSX does not support readlink '-f' flag, work
 # around that
 case $OSTYPE in
     darwin*)
-        SCRIPT=$(readlink $0 || true)
+        SCRIPT=$(readlink "$0" || true)
     ;;
     *)
-        SCRIPT=$(readlink -f $0 || true)
+        SCRIPT=$(readlink -f "$0" || true)
     ;;
 esac
-[ -z $SCRIPT ] && SCRIPT=$0
-SCRIPT_DIR="$(cd `dirname "$SCRIPT"` && pwd -P)"
+[ -z "$SCRIPT" ] && SCRIPT=$0
+SCRIPT_DIR="$(cd "$(dirname "$SCRIPT")" && pwd -P)"
 RELEASE_ROOT_DIR="$(cd "$SCRIPT_DIR/.." && pwd -P)"
 # Make the value available to variable substitution calls below
 export REL_NAME="{{ rel_name }}"
 REL_VSN="{{ rel_vsn }}"
 ERTS_VSN="{{ erts_vsn }}"
 REL_DIR="$RELEASE_ROOT_DIR/releases/$REL_VSN"
-ERL_OPTS="{{ erl_opts }}"
```

Comment:
saw that one removed in a lint-check commit; what about it? was it unused?

Reply:
That is what the linter pointed out. I can't find it used anywhere, and  its not a general arg that `erlexec` will use, as far as I know?, so removed it.